src/frontend/apps/desk/src/features/teams/components/PanelTeam.tsx

```diff
@@ -15,47 +16,80 @@ interface TeamProps {
 export const PanelTeam = ({ team }: TeamProps) => {
   const { t } = useTranslation();
   const { colorsTokens } = useCunninghamTheme();
+  const {
+    query: { id },
+  } = useRouter();
 
   // There is at least 1 owner in the team
   const hasMembers = team.accesses.length > 1;
+  const isActive = team.id === id;
 
   const commonProps = {
     className: 'p-t',
     width: 52,
     style: {
       borderRadius: '10px',
       flexShrink: 0,
+      background: '#fff',
     },
   };
```

Comment:
nit: The border color is non the same as on the Figma design

<img width="467" alt="Capture d’écran 2024-02-19 à 17 49 39" src="https://github.com/numerique-gouv/people/assets/45729124/e7eb59b1-0dcf-47fe-bfc3-b5620a58226c">

Reply:
When the group is empty the color is different.
![image](https://github.com/numerique-gouv/people/assets/25994652/2189255a-4e33-4725-b11a-c1fafb19cc66)

Reply:
I put as well the color lighter, seems better, less button clickable.